src/exe_rules.ml

```diff
@@ -111,8 +111,14 @@ let executables_rules ~sctx ~dir ~dir_kind ~expander
 
 let rules ~sctx ~dir ~dir_contents ~scope ~expander ~dir_kind
       (exes : Dune_file.Executables.t) =
+  let name =
```

Comment:
Maybe we should pass a list here? In this case, the trace should mention `executables foo, bar and baz in src` rather than `executable foo, bar, baz in src`

Reply:
Fixed. I agree that just passing the list of all exes/locs is simpler